### ISSUE TITLE
:trigger support broken with Vue 2.6

### ISSUE THREAD

**Issue by timjenkins** [MikaelEdebro/vue-airbnb-style-datepicker#87]
Upgrading to Vue 2.6 seems to break the trigger boolean functionality. It might be related to their changes to $nextTick: https://github.com/vuejs/vue/issues/9478

One possible solution could be to change usage of `$nextTick()` to `setTimeout()`

**Comment by daamsie** [comment]
I'm having issues with triggering but it's to do with the v-click-outside behavior. I suspect perhaps you have the same problem here. 

I have an input the user clicks into. It has an @click event like so: 

`@click="triggerArrivalDatepicker"`

The triggerArrivalDatepicker sets my trigger value like this

```
triggerArrivalDatepicker(e) {
      this.showingArrivalDatepicker = true;
}
```

the datepicker is set up to trigger based on this value

`:trigger="showingArrivalDatepicker"`

After upgrading from a fairly old version, this logic stopped working. I tracked it down to the v-click-outside. When the input element is clicked, it is outside the calendar so it instantly closes it again. 

I could avoid the problem by changing my trigger function slightly 

```
triggerArrivalDatepicker(e) {
    e.stopPropagation();
    this.showingArrivalDatepicker = true;
}
```

Perhaps try the same to see if it resolves the issue.

**Comment by MikaelEdebro** [comment]
This should be fixed in v 2.5.0. 

@timjenkins thanks for the hint. Seems like there was something done in Vue 2.6 that altered the logic. I wrapped the openDatepicker() call in a setTimeout, and it seems to have done the trick. So e.stopPropagation should not be needed with 2.5.0.
